docs/prover/approx/hashing.md

```diff
@@ -0,0 +1,182 @@
+Modeling of Hashing in CVT
```

Comment:
I prefer to wrap the markdown files to 80 characters for various reasons

Reply:
done